### ISSUE TITLE
Test failure: JIT\\jit64\\hfa\\main\\testE\\hfa_sf2E_r\\hfa_sf2E_r.cmd

### ISSUE THREAD

**Issue by v-haren** [dotnet/runtime#36964]
failed in job: [runtime-coreclr jitstress2-jitstressregs 20200524.1 ](https://dev.azure.com/dnceng/public/_build/results?buildId=658473&view=ms.vss-test-web.build-test-results-tab&runId=20435328&resultId=101232&paneView=debug)

failed tests:
JIT\\jit64\\hfa\\main\\testE\\hfa_sf2E_r\\hfa_sf2E_r.cmd
JIT\\jit64\\hfa\\main\\testE\\hfa_nf2E_d\\hfa_nf2E_d.cmd
JIT\\jit64\\hfa\\main\\testE\\hfa_nd2E_d\\hfa_nd2E_d.cmd
JIT\\jit64\\hfa\\main\\testE\\hfa_nd2E_r\\hfa_nd2E_r.cmd
JIT\\jit64\\hfa\\main\\testE\\hfa_sd2E_d\\hfa_sd2E_d.cmd
JIT\\jit64\\hfa\\main\\testE\\hfa_sf2E_d\\hfa_sf2E_d.cmd
JIT\\jit64\\hfa\\main\\testE\\hfa_sd2E_r\\hfa_sd2E_r.cmd
JIT\\jit64\\hfa\\main\\testE\\hfa_nf2E_r\\hfa_nf2E_r.cmd

Error message
~~~
Return code: 1
Raw output file: C:\h\w\A6980933\w\C03F0A6A\e\JIT\jit64\Reports\JIT.jit64\hfa\main\testE\hfa_sf2E_r\hfa_sf2E_r.output.txt
Raw output:
BEGIN EXECUTION
 "C:\h\w\A6980933\p\corerun.exe" hfa_sf2E_r.dll 
[Identity HFA 01]PASSED
[Identity HFA 02]FAILED => expected = 3, actual = 1
[Identity HFA 03]PASSED
[Identity HFA 05]PASSED
[Identity HFA 08]PASSED
[Identity HFA 11]PASSED
[Identity HFA 19]PASSED
[Get HFA 01]PASSED
[Get HFA 02]FAILED => expected = 3, actual = 1
[Get HFA 03]PASSED
[Get HFA 05]PASSED
[Get HFA 08]PASSED
[Get HFA 11]PASSED
[Get HFA 19]PASSED
Expected: 100
Actual: 1
END EXECUTION - FAILED
FAILED
Test Harness Exitcode is : 1
To run the test:
> set CORE_ROOT=C:\h\w\A6980933\p
> C:\h\w\A6980933\w\C03F0A6A\e\JIT\jit64\hfa\main\testE\hfa_sf2E_r\hfa_sf2E_r.cmd
Expected: True
Actual: False


Stack trace
   at JIT_jit64._hfa_main_testE_hfa_sf2E_r_hfa_sf2E_r_._hfa_main_testE_hfa_sf2E_r_hfa_sf2E_r_cmd() in F:\workspace\_work\1\s\artifacts\tests\coreclr\Windows_NT.x86.Checked\TestWrappers\JIT.jit64\JIT.jit64.XUnitWrapper.cs:line 8579
~~~

**Comment by BruceForstall** [comment]
Failure mode: `CoreCLR Windows_NT x86 Checked jitstress2_jitstressregs4 @ Windows.10.Amd64.Open`

@dotnet/jit-contrib

**Comment by CarolEidt** [comment]
Fixed by #36994 or #37062
